src/kvilib/irc/KviAvatar.cpp

```diff
@@ -51,17 +51,17 @@
 		The user that wants to offer a digital representation of himself
 		will be called "source user". The ones that will receive the notification
 		will be called "target users".
-		Every irc user can be either a source user or target user.
+		Every IRC user can be either a source user or target user.
 		[big]The idea[/big]
-		Every irc user has a client-side property called AVATAR. Let's say that there
+		Every IRC user has a client-side property called AVATAR. Let's say that there
 		are two users: A and B.[br]
 		When user A wants to see the B's avatar he simply sends a CTCP AVATAR request
-		to B (the request is sent through a PRIVMSG irc command).[br]
+		to B (the request is sent through a PRIVMSG IRC command).[br]
 		User B replies with a CTCP AVATAR notification (sent through a NOTICE irc command)
-		with the name or url of his avatar.[br]
+		with the name or URL of his avatar.[br]
 		The actual syntax for the notification is:[br]
 		[b]AVATAR <avatar_file> [<filesize>][/b]
-		The <avatar_file> may be either the name of a B's local image file or an url
+		The <avatar_file> may be either the name of a B's local image file or an URL
```

Comment:
needs to be `a URL`. You pronounce it as `you are el` and since the `y` is considered a consonant in this case, we use `a` instead of `an`.